spec/grape/endpoint_spec.rb

```diff
@@ -1149,6 +1149,39 @@ def memoized
       end
     end
 
+    describe 'delete 200, with a return value (no explicit body)' do
+      it 'responds to /example delete method' do
+        subject.send(:delete, '/example', anchor: false) do
```

Comment:
I don't think you need to `send` these, `delete` is public, so `subject.delete ...`